scripts/globals/treasure.lua

```diff
@@ -1,493 +1,1540 @@
-        };
-    elseif (zone == 150) then -- Monastic Cavern
-        return {
-            dsp.job.DRK,    BORGHERTZ_S_SHADOWY_HANDS,      14221, -- Chaos Flanchard
-            dsp.job.RNG,    BORGHERTZ_S_CHASING_HANDS,      12648, -- Hunter's Jerkin
-            dsp.job.BLM,    BORGHERTZ_S_SORCEROUS_HANDS,    12641, -- Wizard's Coat
-        };
-    elseif (zone == 151) then -- Castle Oztroja
-        return {
-            dsp.job.DRK,    BORGHERTZ_S_SHADOWY_HANDS,      12645, -- Chaos Cuirass
-            dsp.job.BRD,    BORGHERTZ_S_HARMONIOUS_HANDS,   14223, -- Choral Cannions
-            dsp.job.THF,    BORGHERTZ_S_SNEAKY_HANDS,       14219, -- Rogue's Culottes
-            dsp.job.RDM,    BORGHERTZ_S_VERMILLION_HANDS,   12642, -- Warlock's Tabard
-        };
-    elseif (zone == 153) then -- The Boyahda Tree
-        return {
-            dsp.job.NIN,    BORGHERTZ_S_LURKING_HANDS,      13869, -- Ninja Hatsuburi
-        };
-    elseif (zone == 159) then -- Temple of Uggalepih
-        return {
-            dsp.job.SMN,    BORGHERTZ_S_CALLING_HANDS,      12650, -- Evoker's Doublet
-            dsp.job.SAM,    BORGHERTZ_S_LOYAL_HANDS,        13781, -- Myochin Domaru
-        };
-    elseif (zone == 161) then -- Castle Zvahl Baileys
-        return {
-            dsp.job.WAR,    BORGHERTZ_S_WARRING_HANDS,      14214, -- Fighter's Cuisses
-            dsp.job.THF,    BORGHERTZ_S_SNEAKY_HANDS,       12643, -- Rogue's Vest
-        };
-    elseif (zone == 169) then -- Toraimarai Canal
-        return {
-            dsp.job.SMN,    BORGHERTZ_S_CALLING_HANDS,      14103, -- Evoker's Pigaches
-        };
-    elseif (zone == 176) then -- Sea Serpent Grotto
-        return {
-            dsp.job.NIN,    BORGHERTZ_S_LURKING_HANDS,      14101, -- Ninja Kyahan
-        };
-    elseif (zone == 195) then -- The eldieme Necropolis
-        return {
-            dsp.job.BLM,    BORGHERTZ_S_SORCEROUS_HANDS,    14217, -- Wizard's Tonban
-        };
-    elseif (zone == 197) then -- Crawler's Nest
-        return {
-            dsp.job.BRD,    BORGHERTZ_S_HARMONIOUS_HANDS,   13857, -- Choral Roundlet
-            dsp.job.WAR,    BORGHERTZ_S_WARRING_HANDS,      12511, -- Fighter's Mask
-            dsp.job.WHM,    BORGHERTZ_S_HEALING_HANDS,      14216, -- Healer's Pantaloons
-            dsp.job.RNG,    BORGHERTZ_S_CHASING_HANDS,      14224, -- Hunter's Braccae
-        };
-    elseif (zone == 200) then -- Garlaige Citadel
-        return {
-            dsp.job.BST,    BORGHERTZ_S_WILD_HANDS,         12517, -- Beast Helm
-            dsp.job.PLD,    BORGHERTZ_S_STALWART_HANDS,     12515, -- Gallant Coronet
-            dsp.job.WHM,    BORGHERTZ_S_HEALING_HANDS,      13855, -- Healer's Cap
-            dsp.job.MNK,    BORGHERTZ_S_STRIKING_HANDS,     12512, -- Temple Crown
-            dsp.job.RDM,    BORGHERTZ_S_VERMILLION_HANDS,   14218, -- Warlock's Tights
-        };
-    elseif (zone == 205) then -- Ifrit's Cauldron
-        return {
-     
```

Comment:
I blame Zynjec for making me panic.

`
local treasureMinLvl = info.treasureLvl - 10
if mJob ~= dsp.job.THF or mLvl < treasureMinLvl then
`
No need to store treasureMinLvl per zone if it's always 10 levels below the treasureLvl

Reply:
github is fucking drunk

Reply:
Don't blame the messenger, I thought you just hated DRG.